workloads/kube-burner/csv_modifier.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+from typing import *
+import sys
+import argparse
+import os
+
+def get_args() -> argparse.Namespace:
```

Comment:
Should we add this to the main utils folder to be able to be used by all the workloads